lasio/reader.py

```diff
@@ -323,6 +323,10 @@ def determine_section_type(section_title):
         return "Data"
     elif stitle[:2] == "~O":
         return "Header (other)"
+    # las30 transitional code till data parsing is robust for ~A and '_Data'
+    # sections
+    elif stitle.find('_Data') > -1:
+        return "Las30_Data"
```

Comment:
Small point but can we make it `3` instead of `30`?

Reply:
Done!, Made the change in 97a72d9.